src/actions/axios.js

```diff
@@ -11,8 +12,16 @@ const axios = axiosBase.create({ baseURL: baseUrl })
 // them using a page. A possible improvement would be to gracefully prompt
 // them to renew their session, but this will be ok for now.
 axios.interceptors.response.use(null, err => {
-  if (err.response.status === 401) {
-    Router.replaceRoute('login')
+  if (err.response) {
+    // Status code outside of 2xx
+    if (err.response.status === 401) {
+      Router.replaceRoute('login')
+    } else {
+      toast.error(err.response.data)
```

Comment:
For safety, should probably check for null/empty string and show a generic error message if none is in the response.